tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -51,3 +51,27 @@ Feature: full text search
       | dav-path-version |
       | old              |
       | new              |
+
+
+  Scenario Outline: as a sharee search a file tagged by sharee using tag
```

Comment:
please don't start the scenario title with `a` , `the`, `as a` , we don't do that in other scenarios as well and it's good to be consisteint 
```suggestion
  Scenario Outline: sharee searches a file tagged by sharee using tag
```